grapher/lineCharts/LineChart.tsx

```diff
@@ -259,7 +259,7 @@ class Lines extends React.Component<LinesProps> {
     private container?: SVGElement
     componentDidMount(): void {
         const base = this.base.current as SVGGElement
-        const container = base.closest("svg") as SVGElement
+        const container = base.closest("g.LineChart") as SVGElement
```

Comment:
Since this element is in the render tree of the component:

https://github.com/owid/owid-grapher/blob/f0fb2f0352853da969392375e7ce86aaac84604c/grapher/lineCharts/LineChart.tsx#L718

...it might be better to attach event listeners through React, instead of the DOM?